cg/meta/report/rnafusion.py

```diff
@@ -71,8 +71,8 @@ def get_genome_build(self, analysis_metadata: AnalysisModel) -> str:
     def get_report_accreditation(
         self, samples: List[SampleModel], analysis_metadata: AnalysisModel
     ) -> bool:
-        """Checks if the report is accredited or not. Rnafusion is an accredited workflow."""
-        return True
+        """Checks if the report is accredited or not. Rnafusion is not an accredited workflow."""
+        return False
```

Comment:
This function seems to just return false no matter what. It does not seem like a check to me. 

Would it be possible to have an ```is_accredited``` value somewhere in the I suppose RNAfusion ```AnalysisModel ```?

Reply:
So, for other workflows accreditation depends on other factors and not only in the workflow. For example, in balsamic some panels and analysis type are accredited and others not. I don't feel confident changing the logic only in this pipeline. However, it could be good to revisit this part of the code and refactor to make it clearer but for that I would wait until @ivadym is back as he understand the delivery report logic much better.

Reply:
`is_accredited` is a better name for that method, I agree. I'll refactor it in the future. Thanks!